### PR TITLE
Add new Actions to Request a new Team

### PR DESCRIPTION
**Pull request by ghost**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] ~~I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation.~~ -> Not applicable, because its an update of an existing connector
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

File: certified-connectors/Teams-Spirit/apiProperties.json [hunk 1]
```diff
@@ -1,53 +1,55 @@
...
-            },
-            "tenantId": {
-              "value": "common"
+    "properties": {
+      "connectionParameters": {
+        "token": {
+          "type": "oauthSetting",
+          "oAuthSettings": {
+            "identityProvider": "aad",
+            "clientId": "{clientId}",
```
Comment (vmanoharas) [inline certified-connectors/Teams-Spirit/apiProperties.json]: @tillkaemmerer can you please provide dummy string as value, like below. 
"clientId": [[DUMMY]]

File: certified-connectors/Teams-Spirit/apiProperties.json [hunk 2]
```diff
@@ -1,53 +1,55 @@
...
+            "scopes": [
+              "api://app.teams-spirit.com/3a57026a-cf3f-4295-93f5-ca245af3d2df/access_as_user"
+            ],
+            "redirectMode": "Global",
```
Comment (vmanoharas) [inline certified-connectors/Teams-Spirit/apiProperties.json]: Hi @tillkaemmerer, 

Can you please update redirectMode as "GlobalPerConnector"?

File: certified-connectors/Teams-Spirit/apiProperties.json [hunk 3]
```diff
@@ -1,53 +1,55 @@
...
-    "capabilities": [],
-    "publisher": "D.F.K. Digital Teamwork GmbH",
-    "stackOwner": "D.F.K. Digital Teamwork GmbH"
-  }
-}
\ No newline at end of file
+      },
+      "iconBrandColor": "#e7e7ef",
+      "capabilities": [
+
```
Comment (vmanoharas) [inline certified-connectors/Teams-Spirit/apiProperties.json]: Can you please remove this space?

**Review by vmanoharas — COMMENTED** [review]
Hi @tillkaemmerer, 

Can you please resolve review comments?

**Comment by ghost** [comment]
@vmanoharas I implemented the requested changes.

**Comment by vmanoharas** [comment]
Hello @tillkaemmerer, 

Could you please sync yourrepo with main repo. Then, reopen the PR again with latest commit with your changes, as our build pipeline is failing.

**Comment by ghost** [comment]
@vmanoharas Sorry for the delay. I synced my repo. I don't think it is necessary to reopen a PR, because the sync influenced this PR.

**Comment by vmanoharas** [comment]
Hi @tillkaemmerer, Still swagger and api properties validations are failing. Please close this PR and resync your repo with main repo and create a new PR.

**Pull request closed without merging**